src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/VulnerabilityNotificationMessageBuilder.java

```diff
@@ -148,14 +148,13 @@ public String getNotificationType() {
         for (VulnerabilitySourceQualifiedId vulnerabilityItem : vulnerabilityList) {
             String vulnerabilityId = vulnerabilityItem.getVulnerabilityId();
             String vulnerabilityUrl = vulnerabilityItem.getVulnerability();
-
             LinkableItem vulnIdItem = new LinkableItem(MessageBuilderConstants.LABEL_VULNERABILITIES, vulnerabilityId, vulnerabilityUrl);
-            // TODO in blackduck-common:45.1.2+ the VulnerabilitySourceQualifiedId includes the severity so we don't need to make a request to get it
-            String notificationSeverity = blackDuckResponseCache.getSeverity(vulnerabilityUrl);
-            if (vulnerabilityFilters.isEmpty() || vulnerabilityFilters.contains(notificationSeverity)) {
+
+            String notificationSeverity = vulnerabilityItem.getSeverity();
```

Comment:
We could do `Objects.requireNonNullElse(vulnerabilityItem.getSeverity(), "UNKNOWN")`